vm/src/object/gc/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod object;
+mod traverse;
```

Comment:
though traverse is only used for gc, I think traverse itself is a bit out of gc's implementation detail.
I'd like to suggest `vm/src/object/traverse.rs`.
it will be also helpful to enable traverse code even when disabling gc